applications/GeoMechanicsApplication/custom_elements/line_interface_element.cpp

```diff
@@ -264,13 +231,10 @@ std::vector<Matrix> LineInterfaceElement::CalculateLocalBMatricesAtIntegrationPo
 
 std::vector<double> LineInterfaceElement::CalculateIntegrationCoefficients() const
 {
-    auto calculate_integration_coefficient = [p_policy = mStressStatePolicy.get()](
-                                                 const auto& rIntegrationPoint, auto DetJ, const auto& rGeometry) {
-        return p_policy->CalculateIntegrationCoefficient(rIntegrationPoint, DetJ, rGeometry);
-    };
-
-    auto result = ::CalculateIntegrationCoefficients(
-        mIntegrationScheme->GetIntegrationPoints(), GetGeometry(), calculate_integration_coefficient);
+    const auto determinants_of_jacobian = CalculateDeterminantsOfJacobiansAtIntegrationPoints(
+        mIntegrationScheme->GetIntegrationPoints(), GetGeometry());
+    auto result = mpIntegrationCoefficientsCalculator->Run<>(
+        mIntegrationScheme->GetIntegrationPoints(), determinants_of_jacobian, this);
 
     return result;
```

Comment:
Because of your changes, the function got so simple we could just immediately return 
```suggestion
    return mpIntegrationCoefficientsCalculator->Run<>(
        mIntegrationScheme->GetIntegrationPoints(), determinants_of_jacobian, this);
```

Reply:
done